tensorboard/webapp/tb_polymer_interop_types.ts

```diff
@@ -13,34 +13,27 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 import {FeatureFlags} from './feature_flag/types';
+import {TfGlobals, TfStorage} from './tb_polymer_interop_type_definitions';
 
 declare global {
   // createElement type uses the TagNameMap underneath and returns the right type.
   interface HTMLElementTagNameMap {
     'tf-backend': TfBackendElement;
-    'tf-globals': TfGlobalsElement;
     'tf-feature-flags': TfFeatureFlagsElement;
-    'tf-storage': TfStorageElement;
     'tf-paginated-view-store': TfPaginatedViewStoreElement;
     'vz-histogram-timeseries': VzHistogramTimeSeriesElement;
   }
-}
-
-export declare interface TfGlobals {
-  setUseHash(use: boolean): void;
-}
 
-export declare interface TfGlobalsElement extends HTMLElement {
-  tf_globals: TfGlobals;
+  interface Window {
```

Comment:
Is this redundant with tb_polymer_interop_type_definitions?

Reply:
For some reason the polymer binary doesn't recognize the global definition (this is why it needs to import  `tb_polymer_interop_types`). That means that without this definition here the polymer binary doesn't recognize that this type exists.

Reply:
I patched in your PR locally and deleted this `interface Window` block. Everything seems to compile and run. I didn't try importing it internally, though. If you still think there is some reason to duplicate it here, best leave a comment.